### ISSUE TITLE
Cheap power at night time - minning

### ISSUE THREAD

**Issue by ltwarrior18** [bzminer/bzminer#167]
Hello I would love to get settings to create a weekly schedule by hour, when what coin to mine using what overclock settings. Some people have 4 time zones, I have 2 time zones: day time, which is more expensive, and night time, which is cheaper. I would love to maybe even skip day time mining, and mine only at night time, it will increase the efficiency for me as the cost to mine is lower, but I will have 40% less output, which is fine fore me.  Or let us say mine at daytime kaspa+zil with the most efficient settings available, than switch at night time to mine kaspa+etc+zil.
Also Saturdays and Sundays in my case are considered night time tariff.

This would allow me to optimize my costs, increase profits, and keep mining.  Also power prices will keep continue to grow, so this feature would help a lot. Even for solar miners, they have specific hours when power might be very cheap due to sun position at current time. No everybody can afford batteries to store energy to line up the saving equally for the rest of the day, and also efficiency drops.

I am also considering to buy digital timers, for asic miners, so they would turn on and off on specific times, but I don't want to turn off my pcs with gpus.

**Comment by wizard92s** [comment]
I think you could do this using Task Scheduler in Windows, you can learn more by simply searching Windows Task Scheduler on Google.

First you have to create your .bat file for every coin and overclock you want, then you could create various tasks for each hours and days you want.

For example:
kaspa+zil start on 8.00am and stop on 6.00pm, from Monday to Friday
kaspa+etc+zil start on 6.00pm and stop on 8.00am, from Monday to Thursday
kaspa+etc+zil start on 6.00pm on Friday and stop on 8.00 on Monday

With this you can cover the whole week by mining what you want ad desired OCs.
For ASICs the only way is to attach a digital timers or a WiFi controlled socket with scheduling functions (like Tapo).
Maybe you can do it with Flightsheets in HiveOS too (for RIGs and ASICs), but I wouldn't know where to start.

**Comment by ltwarrior18** [comment]
Yes I have been pointed, that you can do with hiveos or task scheduler in windows by other users too, but what I am requesting is to just integrate same thing into the miner level, instead of going around the issue. I really love how bzminer has so many commands for different algos, algo overclocks, timer to switch algos.  Adding additional options, I could create even more complexity, and I would use only 1 single platform, which simplifies the process.
